### ISSUE TITLE
Events - add support for eventType and createdAt

### ISSUE THREAD

**Issue by tiblu** [citizenos/citizenos-api#3]
Events - add support for eventType and createdAt.

New fields:
    * createdAt (or similar) that can be modified via API
    * eventType - free text, partner classificator
`GET /topics/:topicId` - include=events - ideally also provide in what detail you want the event object.

https://github.com/rahvaalgatus/rahvaalgatus/issues/63

**Comment by moll** [comment]
I recommend just naming the column `type` and not `eventType` for it to not be redundant. :)

**Comment by tiblu** [comment]
@moll Sounds good, updated the description.

**Comment by anettlinno** [comment]
@ilmartyrk could you please share your thoughts about how this would be relevant to the follow-up phase? Thank you in advance.

**Comment by ilmartyrk** [comment]
Currently one have only text input  and simple list of items in the follow-up phase view. We could possibly add different kind of results or events under there. For example rahvaalgatus.ee also uses it to add milestones of voting, that doesn't mean that we would have to do the same. But we could have different categories: For example if the topic was call to action, user could insert that some task got done. If it was a legal change, it could display if the proposal was rejected or accepted etc.

**Comment by kevincrepin** [comment]
Yes, I like the idea of having more "mechanics" available for the follow-up phase to emphasise on certain points.

**Comment by anettlinno** [comment]
These would be great additions for the follow-up phase! We have thought about how to empower people to take action after the discussion. If people could use our platform for discussions, collaborative decision-making AND ACTIONS, it would extend the value chain for users. Some ideas from users feedback (and what @ilmartyrk proposed :))

- **Opportunity to add events** (what @ilmartyrk described above) with location, time, short description and host contacts (both online and offline events). User case: Biotoopia was raising discussion about bioharmony which was open for two weeks. Discussion ended with ZOOM-event where the arguments from COS online-discussion were considered in. As a third step they held a conference;
- Call-to-action for volunteers, opportunity to **register people**. User case: there has been discussion about getting premises for Village House from local government. People proposed this idea for the local government which provided them one old building and some resources to restore it. In COS follow-up people could register themselves to become volunteers in repairing the old building for the Village Center;
- **Fundraising** for community project. User case: people wanted to build skate park for their children. They discussed this on our platform, voted over where to build it and got green light from the local government which committed to support with half of the money they needed. The other half should be crowdfunded and now they need to fundraise money for their project;
- **Task list** (as Ilmar described above) to agree and check on the things that are needed to do;
- **Legal change proposal** (as Ilmar described above) could track the proposal´s life cycle;

I know that some of these might be out of our scope (like fundraising), but I thought to leave the mark here. Anything else? :)

**Comment by anettlinno** [comment]
Triage 56. We don´t need to solve all the issues inside our platform but we can redirect users in follow-up phase to continue with further actions (fundraising, registration etc). Sending back to In Preparation for further discussion about how we could empower people to continue with actions after they have discussed and voted over the topics.
